dvtm.c

```diff
@@ -939,6 +953,14 @@ getshell(void) {
 	return "/bin/sh";
 }
 
+static bool
+set_blocking(int fd, bool blocking) {
+	int flags = fcntl(fd, F_GETFL, 0);
+	if (flags < 0) return false;
+	flags = blocking ? (flags&~O_NONBLOCK) : (flags|O_NONBLOCK);
+	return (fcntl(fd, F_SETFL, flags) == 0) ? true : false;
```

Comment:
My C is rusty but it appears we can omit the ternary operator here.

Reply:
Good catch! I copy pasted that code so not surprising.